connect-examples/server-library-example/README.md

```diff
@@ -85,7 +85,7 @@ reading, writing and processing data in the custom format. The plugins (`CustomC
 4. **Copy relevant JARs to the root of the unpacked Spark distribution**:
    ```bash
     cp \
-    <SPARK_HOME>/connect-examples/server-library-example/resources/spark-daria_2.13-1.2.3.jar \
+    <SPARK_HOME>/connect-examples/server-library-example/common/target/spark-daria_2.13-1.2.3.jar \
```

Comment:
Because the spark-daria_2.13-1.2.3.jar removed and package spark-daria_${scala.binary}-1.2.3.jar. How about using `spark-daria_${scala.binary}-1.2.3.jar` instead?

Reply:
It will be `spark-daria_2.13-1.2.3.jar` anyway so the current change makes sense. I will take a separate look for those connect examples. There are few PRs going on, and I would like to avoid conflicting it if you don't mind.

Reply:
Sounds good to me.